### ISSUE TITLE
Windows machine image

### ISSUE THREAD

**Issue by akutz** [kubernetes-sigs/image-builder#30]
Per @rhockenbury and https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/issues/525:

> K8s windows node support has been GA since 1.14. Are there plans to produce a CAPV windows machine image?

**Comment by jsturtevant** [comment]
I have started working on this for the Windows Proposal in CAPI: https://github.com/kubernetes-sigs/cluster-api/pull/3616

**Comment by jsturtevant** [comment]
I just noticed this is not just windows image but specifically for CAPV. I have added the configuration scripts which would enable CAPV in #382 but did this for Azure packer configuration first. 

Should this issue be considered for Configuration or keep is specifcally for CAPV?  If so would some one familiar with CAPV be able to take a look at the PR to see if it meets needs of CAPV?  I am happy to work with someone to make sure the configuration I started will work for CAPV.
